### ISSUE TITLE
Handle circular import dependencies

### ISSUE THREAD

**Issue by naddison36** [naddison36/sol2uml#96]
sol2uml currently fails to generate diagrams for the `Monaco` contract that is used in the [0xMonaco](https://0xmonaco.ctf.paradigm.xyz/) on-chain pvp racing game.

The problem is the `Monaco.sol` file imports the `Car.sol` file with `import "./cars/Car.sol";` and the `Cars.sol` file has a circular import back to the `Monaco.sol` file with `import "../Monaco.sol";`.

```
sol2uml -b Monaco .    
RangeError: Maximum call stack size exceeded
    at /workspaces/sol2uml/lib/associations.js:7:36
    at Array.find (<anonymous>)
    at findAssociatedClass (/workspaces/sol2uml/lib/associations.js:7:31)
    at findImplicitImport (/workspaces/sol2uml/lib/associations.js:69:58)
    at findAssociatedClass (/workspaces/sol2uml/lib/associations.js:31:12)
    at findImplicitImport (/workspaces/sol2uml/lib/associations.js:69:58)
    at findAssociatedClass (/workspaces/sol2uml/lib/associations.js:31:12)
    at findImplicitImport (/workspaces/sol2uml/lib/associations.js:69:58)
    at findAssociatedClass (/workspaces/sol2uml/lib/associations.js:31:12)
    at findImplicitImport (/workspaces/sol2uml/lib/associations.js:69:58)
```

**Comment by naddison36** [comment]
Fixed for the 2.1.9 release
![Monaco](https://user-images.githubusercontent.com/6491112/185818559-e63df371-314f-46cd-8b40-49e0a6ee30ba.png)

**Comment by naddison36** [comment]
Fixed in https://github.com/naddison36/sol2uml/releases/tag/v2.1.9
